src/main/java/orion/task/Task.java

```diff
@@ -0,0 +1,96 @@
+package orion.task;
+
+/**
+ * Represents a task with a unique ID, description, and completion status.
+ * 
+ * <p>
+ * This is an abstract class that provides the common properties and methods
+ * for all types of tasks, such as TODOs, DEADLINEs, and EVENTs. Specific task
+ * types will extend this class and implement the {@code getTypeIcon} method
+ * to return their respective type icons.
+ * </p>
+ */
+public abstract class Task {
+    private final int taskID;
+    private final String description;
+    private boolean completed;
```

Comment:
Should this be changed to 'isComplete' or something similar to more closely adhere to the coding standard?